include/js/general.js

```diff
@@ -7303,3 +7312,52 @@ $(function () {
 
 	new cbAccordion($('#cbmenu'), true);
 });
+
+
+function findParents(el) {
+	let parents = [];
+	while(el.parentNode) {
```

Comment:
please [read our guidelines](https://discussions.corebos.org/documentation/doku.php?id=en:devel:developmentguidelines) on how to format the code